solidity/dashboard/src/components/Message.jsx

```diff
@@ -11,6 +11,7 @@ import PendingActionMessage from "./messages/PendingActionMessage"
 import SuccessMessage from "./messages/SuccessMessage"
 import ErrorMessage from "./messages/ErrorMessage"
 import DelegationAlreadyCopiedMessage from "./messages/DelegationAlreadyCopiedMessage"
+import TopUpReadyToBeCommittedMessage from "./messages/TopUpReadyToBeCommittedMessage"
```

Comment:
**TODO** something that i should have done in https://github.com/keep-network/keep-core/pull/2272 but forgot 🤭 :

Move all messages to `index.js` file to reduce the imports.

Not super important tho - we can do it in separate PR after merging this one.

Reply:
> we can do it in separate PR after merging this one.
yeah, let's address it in a separate PR.